pokemon-showdown

```diff
@@ -44,8 +27,7 @@ if (!process.argv[2] || /^[0-9]+$/.test(process.argv[2])) {
 	// host on port 9000:
 	// $ ./pokemon-showdown 9000
 
-	if (!built) build();
```

Comment:
Can you benchmark the performance impact on playing battles? Like if I do `cat inputlog.txt | ./pokemon-showdown simulate-battle` a few times before and now?

Reply:
Running one inputlog through `simulate-battle` takes about 3.3 times as long with ts-node (~2.3 seconds vs ~0.7 seconds). This makes sense, though; ts-node transpiles the code each time PS starts, resulting in a high upfront cost to starting up.

Reply:
That's... worrying. Maybe we should also keep the old build process around for the command-line tools, so it's just the server that uses `ts-node`?

Reply:
Sure, I'm good with that (although I think it might be worth switching to `tsc` sometime for both that and the NPM package).

Reply:
Agreed.